codefori/vscode-ibmi
# Extend the base API to allow other extensions to add their own components.

### Changes

Adds brand new export to the base API which allows other extensions to add their own installable components.

Components are checked on connection to ensure they are installed on the OS.

### How to test this PR

Examples:

1. Run the test cases
2. Ensure extension starts up with existing components as usual.

### Checklist
